model.js

```diff
@@ -21,4 +21,19 @@ exports.fetchReviewsById = (inputId) => {
   });
 };
 
+exports.fetchReviews = () => {
+  return db
+    .query(
+      `SELECT reviews.review_id, reviews.title, reviews.designer, reviews.owner, reviews.review_img_url,
+  reviews.category, reviews.created_at, reviews.votes,
+  COUNT(comments.review_id) AS comment_count FROM reviews
+      LEFT JOIN comments ON reviews.review_id = comments.review_id
+      GROUP BY reviews.review_id
+      ORDER BY reviews.created_at;`
```

Comment:
At the end of this query string, please specify the default order to be descending.